Button/Button.less

```diff
@@ -122,7 +122,7 @@
 
 			&,
 			.client {
-				color: @agate-button-selected-color;
```

Comment:
Was this intentional? Switching from "selected" to "highlighted" for the "selected" state? [Line 118](https://github.com/enactjs/agate/pull/89/files#diff-4caeea3683cfa034336602bc8f94573fL118)